megamek/i18n/megamek/client/messages.properties

```diff
@@ -285,6 +285,7 @@ BoardSelectionDialog.UpdateSize=Update Size
 BoardSelectionDialog.Updating=Updating...
 BoardSelectionDialog.ViewGameBoard=Preview Game Map... *
 BoardSelectionDialog.ViewGameBoardTooltip=Shows a preview of the game's map.<br> * Note: For generated maps and surprise maps the preview is only a sample<br> and will not be the same as the actual map used.
+BoardSelectionDialog.ViewGameBoardButton=refresh preview
```

Comment:
Since this is a button text I feel this needs at least a capital letter as the first